lib/suppressions.cpp

```diff
@@ -274,6 +274,23 @@ bool Suppressions::isSuppressed(const Suppressions::ErrorMessage &errmsg)
     return false;
 }
 
+void Suppressions::dump(std::ostream & out)
+{
+	out << "  <suppressions>" << std::endl;
+	for (std::list<Suppression>::const_iterator it = _suppressions.begin(); it != _suppressions.end(); ++it) {
+		const Suppression &suppression = *it;
+		out << "    <suppression";
+		out << " errorId=\"" << ErrorLogger::toxml(suppression.errorId) << '"';
+		if (!suppression.fileName.empty()) 
+			out << " fileName=\"" << ErrorLogger::toxml(suppression.fileName) << '"';
+		out << " lineNumber=\"" << suppression.lineNumber << '"';
```

Comment:
if there is no lineNumber I think it's better that this is skipped. It's a Cppcheck implementation detail what value it has then.. that we don't need to know about in the addons. It is more pythonic if the `lineNumber` is `None` I think.